src/Sulu/Bundle/AdminBundle/Admin/Admin.php

```diff
@@ -37,21 +40,11 @@ public function configureNavigationItems(NavigationItemCollection $navigationIte
     {
     }
 
-    /**
-     * Returns all the security contexts, which are available in the concrete bundle.
-     *
-     * @return mixed[]
-     */
     public function getSecurityContexts()
     {
         return [];
     }
 
-    /**
-     * Returns all the security contexts, which are available in the concrete bundle.
-     *
-     * @return array
```

Comment:
and here all other classes extends and so have the correct type  no need for a phpstan-type, we should keep this simple.